### DISCUSSION TITLE
Future cost endpoint

### DISCUSSION THREAD

**Discussion by IBBERN** [NREL/REopt-API-Analysis#90] [8 - API: Application Programming Interface]
Hi,
I am trying to query the results of a started future cost calculating job. But I will every time an error message on get the result data, like this: 
`<html>

<head>
	<title>Request Rejected</title>
</head>

<body>An error has occurred. Please consult with your
	administrator.<br><br>Your support ID is: 5180956909710172213<br><br><a href='javascript:history.back();'>[Go
		Back]</a></body>

</html>`

I have started the calculation run with a valid run uuid and the scenario status is although optimal. I have checked it several times. The response of the post was although ok. I have tried these on the NREL server and on my local docker environment with same behavior. 

Is it a issue on querying the data or something in the backend?

Thanks and regards
Ingo

**Comment by Bill-Becker** [comment]
Hi, can you provide the URL that you are using? I suspect you may not be using the /dev URL in place of the /v1, /v2, or /stable URLs. The /futurecosts endpoint is only available in the /dev URL.

**Comment by IBBERN** [comment]
Hi Bill,

I tried these url:
https://developer.nrel.gov/api/reopt/dev/futurecosts/{_uuid_}/results/?api_key={_API_key_}

**Reply by Bill-Becker** [reply]
Hi Ingo, I found the bug! But I'm waiting on our IT department to allow the results URL in our WAF. I don't think this will take longer than 1-2 days, and I'll let you know when it's updated.
